production-dependency-templates/exocom/README.md

```diff
@@ -0,0 +1,53 @@
+# Exocom dependency
+
+_Production plugin for RDS-type dependencies_
+
+## Usage
+- List database `exocom` as a remote dependency in `application.yml`, with `type: exocom`
+  - Populate required `template-config` fields (defined below)
+- List any exocom-specific service data under `dependency-data.exocom` in a service's `service.yml`
```

Comment:
Here I think we should document what exocom expects in the dependency-data (in both the `service.yml` and `application.yml` (message translations). Also the `dependency-data.<id>` is based on the key in app yaml and is up to the user. I think we should use <id> throughout.